### DISCUSSION TITLE
Make Input Form Uppercase

### DISCUSSION THREAD

**Discussion by sofyanlabs** [Budibase/budibase#4716] [Help] [answered]
hello, I want to ask how do I make the input text on the form uppercase only? if i input "a" it will be "A" automatic. 
Thankyou for your help

**Comment by mjashanks** [comment, marked as answer]
Hey @sofyanlabs 

There's no way to **visually** make the input uppercase. However, you can ensure that the value always gets saved in upper case.

In your save button, you should have a `Save Row` action. You can override the necessary column as below:

![image](https://user-images.githubusercontent.com/3524181/155969828-2fb93ccd-978c-4009-b723-a6a37d98a81c.png)

Then, use the handlebars syntax `{{ upcase  Form.Fields.Region }}` (my field is called `Region`). You could do the same with javascript, if you'd prefer.

**Reply by sofyanlabs** [reply]
Thankyou for Help
